core/liquidity/v2/sla.go

```diff
@@ -154,21 +189,28 @@ func (e *Engine) doesLPMeetsCommitment(party string) bool {
 		maxPrice = one.Add(e.priceRange).Mul(midD)
 	}
 
-	notionalVolume := num.DecimalZero()
+	notionalVolumeBuys := num.DecimalZero()
```

Comment:
commenting on line 171/2: couldn't e.orderBook.GetLastTradedPrice() be nil?

separately, should the sla engine interact with the orderbook directly?

Reply:
I honestly have no problem (design wise) with the engine getting data from orderbook. As long as it protected by interface and only get data - not creates orders.